docs/src/Video.doc.js

```diff
@@ -277,6 +281,42 @@ card(
   />
 );
 
+card(
+  <Example
+    name="Video with children"
+    description={`
```

Comment:
can we add a line here to make it clear that `Video` children are _not_ the same as `video` children https://developer.mozilla.org/en-US/docs/Web/HTML/Element/video which expects only `source` or `track` elements. In this gestalt version the children actually refers to overlays "outside" of the html video element in the code - i think this would make it less error prone to users

Reply:
will add it on both prop description and example description.